fvm/accounts_test.go

```diff
@@ -530,10 +489,6 @@ func TestAddAccountKey(t *testing.T) {
 	// Add a single key
 
 	singleKeyTests := []addKeyTest{
-		{
```

Comment:
We've made the decision to roll back the removal of the deprecated key management API, so we should continue to support and test the deprecated API. 

Sorry for the change of plans, we just decided this today.

Reply:
fixed in https://github.com/onflow/flow-go/pull/2344